src/message_pack/token.cr

```diff
@@ -1,5 +1,24 @@
 # :nodoc:
 class MessagePack::Token
+  enum Type
+    Eof
+    Null
+    False
+    True
+    Array
+    Hash
+    Structure
+    Int
+    Uint
+    Float
+    String
+    Binary
+
+    def to_s
+      super.upcase
+    end
```

Comment:
Not sure the .to_s is really necessary if it's only used for debugging purposes, but I put it in for consistency with the previous output.